app/src/main/res/navigation/nav_graph.xml

```diff
@@ -85,7 +85,14 @@
     <action
         android:id="@+id/action_global_historyFragment"
         app:destination="@id/historyFragment" />
-
+    <action
+        android:id="@+id/action_global_synced_historyFragment"
```

Comment:
It's better to make it global, otherwise we have to add the same action to the synced history fragment as well, in order for `HistoryController` to compile.

Reply:
> It's better to make it global, otherwise we have to add the same action to the synced history fragment as well, in order for `HistoryController` to compile.

This is generally the right pattern to follow: a compile-safe way to navigate. The navgraph is there to give us a deterministic way to know how navigation in the app works. Global things should be used sparingly because we now introduce the ability for _any part of the app_ to navigate to a fragment and that can break the fragment we are going to because it may not have the context that is required.

Reply:
@jonalmeida
The only way I could find to compile directions is by duplicating them in both fragments in the grath, that use `HistoryFragment.kt`. Does [this implementation](https://github.com/mavduevskiy/fenix/pull/2/files) looks preferable over having global directions?